### PR TITLE
#7 - Allow businesses to add link to their profiles

### LINKED ISSUE #7: As a business user, I should be able to add links to profile for business endorsements.

### PR DESCRIPTION
**Pull request by asndiallo**

### CODE REVIEW

File: .gitignore
Comment (themellowpea) [inline .gitignore:1]: untrack this file

File: .project
Comment (themellowpea) [inline .project:1]: remove this

File: .vscode/launch.json
Comment (themellowpea) [inline .vscode/launch.json:1]: untrack this folder .vscode

**Review by themellowpea — CHANGES_REQUESTED** [review]
pull from `dev`, resolve merge conflicts. You will most likely have to refactor a few things.

File: .gitignore
Comment (asndiallo) [reply .gitignore:1]: untrack gitignore ?

**Pull request closed without merging**